source/validation.txt

```diff
@@ -37,6 +28,20 @@ specific shape or only allowing a specified range of values in fields.
 .. figure:: /images/compass/validation-view.png
   :figwidth: 720px
 
+Limitations
+-----------
+
+The :guilabel:`Validation` tab is not available: 
+
+- In :guilabel:`Compass Community Edition`. 
+
+- If you are connected to a 
+  `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```

Comment:
We have a replacement for links to the atlas docs. You can do:

```
:atlas:`Data Lake </data-lake>`
```

Reply:
@jeff-allen-mongo fixed; please see stage